website/docs/docs/supported-adapters.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Supported adapters"
```

Comment:
As per Slack, I think "Available adapters" feels like the "correct" heading here.

That being said, I'm also curious what the most common search term is here — I think "supported databases" probably made a ton of sense for SEO purposes, but "available adapters" won't. Hmmmmm

Reply:
Fair, I don't think it's a great step SEO-wise. All in all I'm not too worried about it:
- Searching for a specific supported adapter name ("Redshift," "Presto") still returns useful results
- [This FAQ](https://docs.getdbt.com/docs/about/overview/#what-databases-does-dbt-currently-support) still appears with search terms "database" + "support," and links to the right page (even though the page it's on is unlisted?)
- And I like that this page will now come up with the search term "available"